samples/bluetooth/radio_notification_cb/src/main.c

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/console/console.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <bluetooth/services/latency.h>
+#include <bluetooth/services/latency_client.h>
+#include <bluetooth/gatt_dm.h>
+#include <bluetooth/radio_notification_cb.h>
+
+static struct bt_latency latency;
+static struct bt_latency_client latency_client;
+
+void scan_start(void);
```

Comment:
static?

Reply:
The function is defined in another file, so it should not be declared as static